mscore/playpanel.ui

```diff
@@ -1,557 +1,559 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
  <class>PlayPanelBase</class>
- <widget class="QWidget" name="PlayPanelBase">
+ <widget class="QDockWidget" name="PlayPanelBase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>389</width>
-    <height>293</height>
+    <height>626</height>
    </rect>
   </property>
-  <property name="sizePolicy">
-   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
-    <horstretch>0</horstretch>
-    <verstretch>0</verstretch>
-   </sizepolicy>
+  <property name="floating">
+   <bool>true</bool>
   </property>
-  <property name="focusPolicy">
-   <enum>Qt::ClickFocus</enum>
+  <property name="features">
+   <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
-   <string>Play Panel</string>
+   <string>PlayPanel</string>
```

Comment:
Why this change? I believe this is a user visible string and translatable, and as such should not use CamelCase

Reply:
no reason, it just slipped out, let's keep the former translatable string of course!